packages/aws-cdk/lib/cli/cdk-toolkit.ts

```diff
@@ -143,9 +150,11 @@ export enum AssetBuildTime {
  */
 export class CdkToolkit {
   private ioHost: CliIoHost;
+  private toolkitStackName: string;
 
   constructor(private readonly props: CdkToolkitProps) {
     this.ioHost = props.ioHost ?? CliIoHost.instance();
+    this.toolkitStackName = props.toolkitStackName ?? DEFAULT_TOOLKIT_STACK_NAME;
```

Comment:
Made this a class property to mimic behavior in new toolkit and to provide a fallback when an action option is unset. In practice, the class `toolkitStackName` and action option `toolkitStackName` will be the same in the CLI. Will clean this up in a later follow-up PR.